core/src/main/java/com/graphhopper/GraphHopper.java

```diff
@@ -607,6 +630,7 @@ public GraphHopper init( CmdArgs args )
 
         workerThreads = args.getInt("osmreader.workerThreads", workerThreads);
         enableInstructions = args.getBool("osmreader.instructions", enableInstructions);
+        preferredLanguage = args.get("osmreader.preferred-language", null);
```

Comment:
Here it should be `preferredLanguage = args.get("osmreader.preferred-language", preferredLanguage);` to avoid overwriting possible initialized values